### DISCUSSION TITLE
What do you use to make your diagrams?

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#558] [Q&A]
@rstrouse What software are do you use to make your diagrams?  They are quite impressive.

**Comment by rstrouse** [comment]
Those were all done in Visio.

**Reply by orrious** [reply]
Are you making your own stencils from pictures or already available?  They look clean.

**Reply by rstrouse** [reply]
Making my own.  Everything is just a shape.

**Comment by spifftek70** [comment]
https://app.diagrams.net/
it's free to use and exports diagrams in a variety of formats.

![pool automation-12 drawio-19 drawio-5](https://github.com/tagyoureit/nodejs-poolController/assets/11727190/aaa256d3-6114-42e8-b4c4-b414a125d25f)
